app/scripts/directives/score.directive.js

```diff
@@ -27,13 +27,13 @@ angular.module('scoreTrackerApp')
 
         scope.removeScore = function(uuid) {
           ScoreService.remove(uuid);
-          element.parent().remove();
-          scope.updateSummary();
+          element.remove();
```

Comment:
While it is better to no longer access `parent()`, `element.remove()` is an anti-pattern. Built-in directives like `ng-if` and `ng-repeat` call the remove for you as well as destroying scope. You shouldn't even need this line at all. Once the score object is removed from the `ScoreService`, the `ng-repeat` in the main.html file will react to the change and remove the item for you.